package.json

```diff
@@ -2,6 +2,7 @@
   "name": "jw-ott-webapp",
   "version": "4.19.0",
   "main": "index.js",
+  "type": "commonjs",
```

Comment:
Do we need `commonjs` here if we use modules in our script?

Reply:
Good catch. Yes, it works without it! Thanks for pointing it out.